oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/DocumentStoreIndexerBase.java

```diff
@@ -239,6 +239,12 @@ nodeStore, getMongoDocumentStore(), traversalLog))
         return storeList;
     }
 
+    public IndexStore buildTreeStore() throws IOException, CommitFailedException {
+        System.setProperty(FlatFileNodeStoreBuilder.OAK_INDEXER_SORT_STRATEGY_TYPE,
```

Comment:
This will leave this system property behind, even after finishing building the tree store, so it changes global state. Couldn't this cause unintended problems?

Reply:
I didn't think about that... I'll try to fix it.

I think it's the same way as the "reuse existing flat file store" works btw. we might also revisit these other usages.

Reply:
Fixed here.